why_postgres.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: 'A new serverless Postgres platform'
```

Comment:
Whenever we land on a title, we should probably have the slug match with some discoverable keywords

Reply:
Done now, I used "serverless-data-platform".